.azure-pipelines/update-docs.yml

```diff
@@ -31,8 +31,6 @@ steps:
         R -e 'dir.create(Sys.getenv("R_LIBS_USER"), recursive = TRUE);
               .libPaths(Sys.getenv("R_LIBS_USER"));
               devtools::install_deps(upgrade = FALSE);
-              unlink("man", recursive=TRUE, force=TRUE);
```

Comment:
I already have a PR for this. You can either approve my PR or manually merge the changes to this PR.
https://github.com/Azure/azureml-sdk-for-r/pull/182/files

Reply:
sounds good